packages/rpc-api/src/index.ts

```diff
@@ -188,7 +188,7 @@ export function createSolanaRpcApi<
         parametersTransformer: getDefaultParamsTransformerForSolanaRpc(config) as (params: unknown[]) => unknown[],
         responseTransformer: getDefaultResponseTransformerForSolanaRpc({
             allowedNumericKeyPaths: getAllowedNumericKeypaths(),
-        }),
+        }) as RpcApiConfig['responseTransformer'],
```

Comment:
This is probably because `packages/rpc-transformers/src/response-transformer.ts` is missing a return type annotation. Does the need to do this go away if you add one?